static/styles/banana-styles.css

```diff
@@ -0,0 +1,154 @@
+body {
+    font-family: 'Montserrat', sans-serif !important;
+}
+
+.landing-item {
+    display: flex;
+    flex: 1 1 0px;
+    height: 100%;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    padding: 50px;
+}
+
+.landing-header {
+    font-weight: 900;
+    line-height: 56px;
+    font-size: 55px;
+    padding-right: 5%;
+    color: #2C070B;
+}
+
+.landing-text {
+    padding-right: 10%;
+    font-size: 20px;
+    font-weight: 500;
+    color: #2C0A0A;
+}
+
+.text-highlight {
+    background-color: #F23B4F;
+    color: white;
+    padding-left: 8px;
+    padding-right: 8px;
+    display: inline-block;
+    height: 56px;
+}
+
+@media (max-width: 986px) {
+    .landing-header {
+        line-height: 30px;
+        font-size: 32px;
+        padding-right: 5%;
+        padding-left: 5px;
+    }
+
+    .text-highlight {
+        height: 32px;
+    }
+
+    .landing-text {
+        font-size: 16px;
+    }
+
+    .landing-item {
+        padding: 20px;
+    }
+}
+
+.landing-section {
+    height: 780px;
+}
+
+.banana-bar {
+    position: absolute;
```

Comment:
I did a bit of playing around with the UI and found that this line breaks how the navbar works on the other pages (`about.html` and `admin.html`). In my testing, the content of the page would end up layered on top of the navbar.

Perhaps you could look into why this occurring and provide a fix? I found that removing this line fixes the navbar behaviour. However it does change the look of the main page by extending the navbar to the full width so if that's not what you were going for you may need to devise another solution.